### PR TITLE
Create istio namespace before install remote cluster.

### PR DESCRIPTION
**Pull request by gyliu513**
/cc @sdake @linsun

### CODE REVIEW

File: _docs/setup/kubernetes/multicluster-install.md [hunk 1]
```diff
@@ -147,6 +147,10 @@ $ export STATSD_POD_IP=$(kubectl -n istio-system get pod -l istio=statsd-prom-br
    $ helm template install/kubernetes/helm/istio-remote --name istio-remote --set global.pilotEndpoint=${PILOT_POD_IP} --set global.policyEndpoint=${POLICY_POD_IP} --set global.statsdEndpoint=${STATSD_POD_IP} > $HOME/istio-remote.yaml
    ```
 
+1. Create a namespace for remote istio cluster.
```
Comment (geeknoid) [inline _docs/setup/kubernetes/multicluster-install.md]: for the remote Istio

Comment (gyliu513) [reply _docs/setup/kubernetes/multicluster-install.md]: Done

**Comment by gyliu513** [comment]
@geeknoid @linsun can we have this merged?

**Pull request merged**